README.md

```diff
@@ -61,39 +62,39 @@ func main() {
 
 Requirements:
 
-* Hazelcast Go client is compatible only with Hazelcast IMDG 4.x and above.
-* We support Go 1.15 and up.
+* Hazelcast Go client is compatible only with Hazelcast 4.x and 5.x.
+* We support two most recent releases of Go, currently 1.16 and 1.17.
 
 In your Go module enabled project, add a dependency to `github.com/hazelcast/hazelcast-go-client`:
 ```shell
-# Depend on a specific release
-$ go get github.com/hazelcast/hazelcast-go-client@v1.1.1
+# Depend on the latest release
+$ go get github.com/hazelcast/hazelcast-go-client@latest
 ```
 
 ## Quick Start
 
-Hazelcast Go client requires a working Hazelcast IMDG cluster.
-This cluster handles the storage and manipulation of the user data.
+Hazelcast Go client requires a working Hazelcast cluster.
 
-A Hazelcast IMDG cluster consists of one or more cluster members.
-These members generally run on multiple virtual or physical machines and are connected to each other via the network.
-Any data put on the cluster is partitioned to multiple members transparent to the user.
-It is therefore very easy to scale the system by adding new members as the data grows.
-Hazelcast IMDG cluster also offers resilience.
-Should any hardware or software problem causes a crash to any member, the data on that member is recovered from backups and the cluster continues to operate without any downtime.
+Check out our [Get Started](https://hazelcast.com/get-started/) page for options.
 
-The quickest way to start a single member cluster for development purposes is to use our Docker images.
+### Starting the Client with Hazelcast Cloud
 
-```
-docker run --rm --name hazelcast -p 5701:5701 hazelcast/hazelcast:4.2
-```
+You only need the cluster name and Hazelcast cloud token to start the client.
+If you haven't already, you can [http://cloud.hazelcast.com/sign-up](sign up) for a free Hazelcast Cloud account.
```

Comment:
Link and text should be swapped here. 
```suggestion
If you haven't already, you can [sign up](http://cloud.hazelcast.com/sign-up) for a free Hazelcast Cloud account.
```

Reply:
Fixed at: https://github.com/hazelcast/hazelcast-go-client/pull/710/commits/dc677bec1534ce600c73b48c17ee49fadb3e7eae